src/masked-input/masked-input.style.tsx

```diff
@@ -39,3 +42,71 @@ export const IconContainer = styled.div<IconProps>`
         height: 1.125rem;
     }
 `;
+
+// -----------------------------------------------------------------------------
+// LOADING DISPLAY
+// -----------------------------------------------------------------------------
+export const LoadingWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    height: 3rem;
+`;
+
+export const LoadingLabel = styled(Text.Body)`
+    color: ${Color.Neutral[3]};
+`;
+
+export const Spinner = styled(ComponentLoadingSpinner)`
+    margin-right: 0.5rem;
+    #inner1,
+    #inner2,
+    #inner3,
+    #inner4 {
+        border-color: ${Color.Neutral[3]} transparent transparent transparent;
+    }
```

Comment:
could the `color` prop on the `ComponentLoadingSpinner` be used instead?

Reply:
Abit mafan since we need to pass the theme prop. Styling it this way is easier.